src/helpers/localStorageHelpers.js

```diff
@@ -1,5 +1,5 @@
 import store from 'store';
-import _ from 'lodash';
```

Comment:
just using one function in that file and in whole project only this place was different.